v1x1-modules/v1x1-modules-channel/pom.xml

```diff
@@ -24,6 +24,7 @@
       <module>v1x1-modules-channel-factoids</module>
       <module>v1x1-modules-channel-uptime</module>
       <module>v1x1-modules-channel-wasm</module>
+      <module>v1x1-modules-channel-voicelog</module>
```

Comment:
This isn't in alphabetical order, see `caster` in the middle of the list next to `factoids`